src/platform/node/metrics.js

```diff
@@ -47,12 +47,15 @@ module.exports = function () {
         errorHandler: () => {}
       })
 
+      client.maxBufferSize = Infinity // set after to prevent flush interval
```

Comment:
Do this so we have to manually flush, but manually flushing so we have better control?

Reply:
Exactly. If set in the options, the library will automatically flush at an interval.

Reply:
:+1: thanks